x-pack/plugins/lens/server/migrations.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { cloneDeep } from 'lodash';
+import { SavedObjectUnsanitizedDoc } from 'src/core/server';
```

Comment:
This import is slightly different as the migrations are now importing types from the server